Unity/Assets/Scripts/UI.cs

```diff
@@ -0,0 +1,95 @@
+using UnityEngine;
+using WumpusCore.Controller;
+using WumpusCore.Topology;
+
+public class UI : MonoBehaviour
+{
+    private Controller controller;
+
+    [SerializeField] 
+    private GameObject cam;
+
+    private const float camSens = 5f;
```

Comment:
This const is not following standard C# style standards, use PascalCase

Reply:
wow I am being mean today

Reply:
okkkkkk......... fixed